spec/Schema.spec.js

```diff
@@ -178,6 +178,38 @@ describe('SchemaController', () => {
     });
   });
 
+  it('class-level permissions test count', (done) => {
+    var obj;
+    return config.database.loadSchema()
+      // Create a valid class
+      .then(schema => schema.validateObject('Stuff', {foo: 'bar'}))
+      .then(schema => {
+        var count = {};
+        return schema.setPermissions('Stuff', {
+          'create': {'*': true},
```

Comment:
That was to match the style of the rest of the file, we'll update those at a later time (i.e. test linting)